### DISCUSSION TITLE
recipes/ingredients/roasted-garlic/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1172] [Comments]
# recipes/ingredients/roasted-garlic/

A collection of my recipes.

https://nicholaswilde.io/recipes/ingredients/roasted-garlic/

**Comment by nicholaswilde** [comment]
Sprinkle a pinch of salt before roasting.

**Reply by nicholaswilde** [reply]
https://www.tastingtable.com/1666404/better-way-roast-garlic-using-pan/
